kyverno-policy-reporter-ui.yaml

```diff
@@ -34,25 +38,28 @@ pipeline:
   - working-directory: frontend
     runs: |
       bun install
-      bun run build
-
-      mkdir -p ${{targets.destdir}}/app
-      cp -R ./dist ${{targets.destdir}}/app/dist
+      bun run generate
+      mkdir -p ${{targets.contextdir}}/var/run/ko/templates
+      cp -R ./dist ${{targets.contextdir}}/var/run/ko/ui
 
   - uses: go/build
     with:
       modroot: backend
       packages: .
       output: policyreporter-ui
 
+  - name: copy more frontend requirements
+    runs: |
+      cp -R backend/templates ${{targets.contextdir}}/var/run/ko/
+
 subpackages:
   - name: kyverno-policy-reporter-ui-compat
     description: compatibility symlinks package for policyreporter-ui Dockerfile
     pipeline:
       - runs: |
-          # Symlink the binary from usr/bin to /app/
-          mkdir -p ${{targets.subpkgdir}}/app
-          ln -sf /usr/bin/policyreporter-ui ${{targets.subpkgdir}}/app/policyreporter-ui
+          # Symlink the binary from usr/bin to /ko-app/
+          mkdir -p ${{targets.contextdir}}/ko-app
+          ln -sf /usr/bin/policyreporter-ui ${{targets.contextdir}}/ko-app/policyreporter-ui
 
```

Comment:
In tests, can we spin up this on kwok and port-forward to assert that it returns 2xx response code?

Reply:
thank you very much for the review, extended tests via https://github.com/wolfi-dev/os/pull/46006/commits/1ea680340bdbe45e2a8130e5427d22831b7ffede